shell/platform/fuchsia/flutter/vsync_waiter.cc

```diff
@@ -50,6 +52,15 @@ VsyncWaiter::VsyncWaiter(std::string debug_label,
             std::make_unique<fml::WeakPtrFactory<VsyncWaiter>>(this);
       }));
   session_wait_.set_handler(wait_handler);
+
+  std::string* json_string = new std::string();
```

Comment:
are we leaking this string? why not:
"
std::string json_string;
dart_utils::ReadFileToString("/config/data/flutter_engine_config", &json_string);
"
instead?

Reply:
Done